spec/02-integration/04-admin_api/21-client_certificate.lua

```diff
@@ -0,0 +1,226 @@
+YDBeMA4GA1UdDwEB/wQEAwIFoDATBgNVHSUEDDAKBggrBgEFBQcDATAfBgNVHSME
+GDAWgBT0/pI/aP5pnsFvVaZscKVH/A/EBDAWBgNVHREEDzANggtleGFtcGxlLmNv
+bTANBgkqhkiG9w0BAQsFAAOCAYEASx4Mefq4qZBSDTaG62Pc3pEObj3rGWlkz0MV
+AzMA0Zhud5eCU8b81h+p2rzFic0Sy1A3iPNeAuAuaiTwS8Ci/6xwFcrHAzFioZYF
+UCylQ1Lm5J0LGG8Nk6fnFjPLmCUd9qFBnge5zQHQjcfiTe6WZ6Yz2hOEtO3N4bEm
+4Vvr2fzJhso98Ans6pI0VSXdVC4iSCdAbCYi2BAQF9b6iC5/Rc5RqbMDp0Captzm
+WWdo92eLqaNJ6L+8rkElM0xkQxSp/x++sk2JI0x2WeaSWENBB7PjU6MBooSp+kmS
+PCIz55H8MmQsjfg/Uu6TA+iP4m4HWhGhjAZxb+KLl6+2x0cU9W6Fcol2+YLiuNUr
+HgMJM3NQ2hc7cJRW1YR5eidJy0ceftZMTr9aCxKaMFmggffCpjDTFqFe+m0v7vFY
+ha0LuiaIKhnGCv8GHTMcyNuhybDwYV8qmw6/SQtoy7wKJt1zDVUHI2oxwLLhHNOe
+wNeyz2Ot1fWqjHBhFYFIVQcV996C
+-----END CERTIFICATE-----
+]]
+
+local new_key = [[
+-----BEGIN PRIVATE KEY-----
+MIIEvQIBADANBgkqhkiG9w0BAQEFAASCBKcwggSjAgEAAoIBAQDLMVcG3LVM6Eqs
+LHXbyje/E8Sxgost3auPVjZ909yJWE4rG01FN9Cig20b1e3YU07775RylAWwlIko
+ryr7l5ePpHbajlijIjFs3xAXcBE/NQTUkltSU5jBOGHf8bys8tY8gIbYRr79zGAY
+jlxKAveN5WizYAEFxQ9brPsEuwXzc0paCyHNqYtQm1WgzFbDlCj+3MMTfEb1xCol
+hG5JLaRtW8q6njH7gIhjwcSuz3Hcfri0qqjSIvSn1CEUCKNUOO1B2ip8Dm86ZZSf
+1jWDxOyIfJctmk+YHYxzispLnbw21iNW726tVQhA5M0LXAkIFUO+lS/Skk6Q/hct
+pZQhmTkNAgMBAAECggEAQkWKa9uKalYNHNC3dAMlYXXSHk9Ek6YL8Ykd5rY15Lnf
+bOHPtva4XnjdGkoQ9BBf333JfgFXRbt8bL+NFlff/+KVPT1WHYbmdere9bYAegwR
+0j8V9+j6FnMMQR03+3SMeVk2Ic4ZNu/0m6263gzPrwEWwAJc8wPNuYnWqTlSfI8V
+hG/hB7TB/Yz3HbWg+rXE84RsT280MG0gzkK9U1dQVGYuPAfiqZt9ZwQ9HjbVjVLu
+20NqBIzFE4/IV/wXVNHrSUdE8GLuYxlO1uKg6h71FDBsEu3lHyx4S0xhADbZIREr
+ZwY1+PCAL3nuUUc2KuEU1Bc2tjBWgasY2kLioxDzRQKBgQDTA+4TjUzuKbbeZBn6
+m7g+lyK9I/lf+hge++erYzF64WKgbD8cuZmS4c3N5b3iH0pUdc6s6c1eERB+DzqG
+RraB9rD0bqFYWinIEqPvDmzxAnkgNHlar9NGpi4Ow5GHvLCY5POI4/fd9dBOPeKG
+IYJ78rXhYinBkIyaeNv7iVQwRwKBgQD2gn560wekurtzKz/2kOisvwUI5UDR4n5Q
+aypwBO/XoVVdlmKkzZuEq4EPDnxP0WpVAkpVRwYgv0JBL0TblvZEf75dftuvH0g4
+MerD6pWQV9NjMhDfGN9EnomhF4qN2PxfH6Ae8voa7XUgBGrANTVzQ28r+kIxgBhB
+UmcdF7SqCwKBgHxQF2S7XLP+LTZtAIUMUgnFnfiRhetN+gHe44erg03iE+lwe+nf
+SaMnlcgT7b6WMoL5eE2T26HsKawqKo9Kf2CnRW5q8nCLhYvRt3QxKmr9yv/3Cfss
+FCrN4dpeypCNz/fzAaoGq1K+6v39JUNh4G5G+XGTFAhtsPeb9fUE3W0nAoGBAPSe
+JfioNSuIib8Nqhl5TwstO0fKh3o/ce1HiwNCQucNl/KY6yZzSeJrdlSEoWNrgkL9
+VNou8qvIX2zkBUaLFbJ/uV56JnIvbrElB7yGqwhjvsbuxcM9oUEsuZJ8E+gS7M1Q
+Ssqrfvj9rn6pI22831+UCVvwqYMJhAg+R0idX5BlAoGAY6lp43yoEBg7+JEvzrYq
+cTbqrL0lo4W9JsDai65lJ/f6QTP1I53vHiUBUamnbuBD/dbn2CWR0vbkM1gbkM6M
+/rEJqU5K54qS+IDxd27YsTy9SdSTE6UakYn/bweVTed2qmnY1SQZUt1KWThwK4xx
+XeHHwxnzI/T07RdxJufNKbQ=
+-----END PRIVATE KEY-----
+]]
+
+
+for _, strategy in helpers.each_strategy() do
+  describe("invalidate client certificate", function ()
+    local admin_client, cert_id
+
+    lazy_setup(function()
+      local bp, _ = helpers.get_db_utils(strategy)
+      local fixtures = {
+        dns_mock = helpers.dns_mock.new(),
+        http_mock = {
+          ssl_response = [[
+            server {
+              server_name test.com;
+              listen 443 ssl;
+
+              ssl_certificate     /etc/aaa.com.cert.pem;
```

Comment:
Can this run normally without the `/etc` certs? I haven't seen the cert files in this PR

(And I feel it is heavy for using real cert files in mock servers, but I don't know more elegant options, may need to investigate more)